ups/cp_verify.table

```diff
@@ -8,5 +8,6 @@ setupRequired(pex_exceptions)
 setupRequired(pipe_tasks)
 setupRequired(utils)
 setupRequired(pipe_base)
+setupRequired(analysis_tools)
```

Comment:
Is this an okay dependency direction?

Reply:
I believe so.  `analysis_tools` depends on a subset of the `cp_verify` dependencies, in addition to `daf_butler`, `skymap`, and `geom`.  Those three only depend on even more fundamental packages, so I think this is safe from circular issues.